frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-external/details-card.tsx

```diff
@@ -79,13 +95,19 @@ export const DetailsCard: React.FC<DashboardItemProps> = ({
         >
           {ocsName}
         </DetailItem>
-        <DetailItem
-          title={t('ceph-storage-plugin~Provider')}
-          error={!!infrastructureError || (infrastructure && !infrastructurePlatform)}
-          isLoading={!infrastructureLoaded}
-        >
-          {infrastructurePlatform}
-        </DetailItem>
+        {link ? (
+          <DetailItem
+            title={t('ceph-storage-plugin~Provider')}
+            isLoading={!secretLoaded}
+            error={!!secretError}
+          >
+            <ExternalLink href={link} text={t('ceph-storage-plugin~RedHat Ceph Storage')} />
+          </DetailItem>
+        ) : (
+          <DetailItem title={t('ceph-storage-plugin~Provider')}>
+            {t('ceph-storage-plugin~RedHat Ceph Storage')}
+          </DetailItem>
+        )}
```

Comment:
Define `CEPH_BRAND_NAME` in `packages/ceph-storage-plugin/src/constants/common.ts`

Reply:
@afreen23 I more issue if we do this way:
```
          <DetailItem
            title={t('ceph-storage-plugin~Provider')}
            isLoading={!secretLoaded}
            error={!!secretError}
          >
           {link ?  <ExternalLink href={link} text={CEPH_BRAND_NAME}} /> :    CEPH_BRAND_NAME
          </DetailItem>
        )}
```
if `link` is `null` it means there is some error retrieving the secret  (say NotFound error), in that case  `DetailItem` will by default display `Not Available` instead of CEPH_BRAND_NAME (.
So, either Tim need to do as he was doing before (two `DetailItem` on basis on ternary condition) or we pass `error={false} && isLoading={false}` always to `DetailItem`.

Reply:
I synced on a call with Tim, the behavior is few customers can and cannot have ceph dashboard.